packages/types/src/primitive/sszTypes.ts

```diff
@@ -1,4 +1,7 @@
 import {ByteVectorType, UintNumberType, UintBigintType, BooleanType} from "@chainsafe/ssz";
+/* eslint-disable no-restricted-imports */
```

Comment:
Note that it might be worth improving `@chainsafe/ssz` exports so that this import doesn't trigger a lint error.